### DISCUSSION TITLE
CPU Architecture Table page not found:

### DISCUSSION THREAD

**Discussion by Olddantrucker** [86Box/86Box#1513] [General]
https://ci.86box.net/job/86Box-Optimized/ 

<html>
<body>
<h2 style="color: rgb(0, 0, 0); font-family: &quot;Times New Roman&quot;; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">HTTP ERROR 404 Not Found</h2>

URI: | /job/86Box-Optimized/
-- | --
404
Not Found
Stapler

<br class="Apple-interchange-newline">
</body>
</html>

0_0

**Comment by ts-korhonen** [comment]
As stated in [the documentation](https://86box.readthedocs.io/en/latest/dev/builds.html#optimized), optimized builds (86Box-Optimized) have been discontinued as of March 18th 2021.
